package.json

```diff
@@ -59,30 +60,41 @@
     "start:playground:mockDuo": "echo \"DEPRECATED: Please run 'yarn start --mock mockDuo --watch' instead.\" && exit 1",
     "start:dev-server": "echo \"DEPRECATED: Please run 'yarn start' instead.\" && exit 1",
     "start": "node scripts/buildtools start",
+    "start:test:app": "webpack serve --config test/app/webpack.config.js",
     "mock:device-authenticator": "dyson playground/mocks/spec-device-authenticator/ 6512"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.3",
+    "@babel/core": "^7.15.5",
     "@babel/plugin-transform-modules-commonjs": "^7.10.1",
     "@babel/plugin-transform-runtime": "^7.10.3",
-    "@babel/preset-env": "^7.10.3",
+    "@babel/preset-env": "^7.15.6",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/register": "^7.15.3",
     "@babel/runtime-corejs3": "^7.10.3",
     "@okta/eslint-plugin-okta": "0.4.0",
     "@okta/okta-auth-js": "^5.5.0",
     "@okta/okta-idx-js": "0.20.0",
     "@sindresorhus/to-milliseconds": "^1.0.0",
+    "@wdio/cli": "^7.14.1",
+    "@wdio/junit-reporter": "^7.15.0",
+    "@wdio/local-runner": "^7.14.1",
+    "@wdio/mocha-framework": "^7.14.1",
```

Comment:
IIRC we have plans to migrate off of mocha to another framework. Do you have a ticket I could follow to track that work?